docs/run/circuit-execution.mdx

```diff
@@ -26,4 +26,25 @@ Therefore, each circuit is equal in duration to the longest circuit in the batch
 
 ## Dynamic repetition rate execution
 
-Some IBM Quantum systems allow for dynamic repetition rate execution. These systems are identified in Qiskit using `backend.configuration().dynamic_reprate_enabled`, and return a value of `True`. On these systems, it is possible to manually set the above idle time by setting the `rep_delay` of the submitted job. One can see from the above figures that by reducing the idle time one can potentially see a greater throughput of circuits on the systems that support dynamic repetition rates. See the the next section on conditional reset for more detailed usage examples.
\ No newline at end of file
+Some IBM Quantum systems allow for dynamic repetition rate execution. These systems are identified in Qiskit using `backend.configuration().dynamic_reprate_enabled`, and return a value of `True`. On these systems, it is possible to manually set the above idle time by setting the `rep_delay` of the submitted job. One can see from the above figures that by reducing the idle time one can potentially see a greater throughput of circuits on the systems that support dynamic repetition rates.
+
+## Specify rep_delay for a primitive job
+
+```python
+# Make sure your backend supports it
+backend = service.least_busy(operational=True, min_num_qubits=100, dynamic_reprate_enabled=True)
+
+# Determine the allowable range
+backend.rep_delay_range
+sampler = Sampler(backend=backend)
```

Comment:
I think we should include the imports for runtime and primitives, or use some indication that this example is pseudocode, like `...` at the top of the codeblock or something

Reply:
done https://github.com/Qiskit/documentation/pull/1345/commits/b52789b4bab7543722a7e422f041e1e8e1b23100